<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg3</class>
 <widget class="QDialog" name="dlg3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract Features</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>22</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="psoPath_te">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="laser_te">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pos_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PosFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lasers_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>LaserFiles</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extract_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>processing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
